examples/python-cicd-with-cloudbuilder/cloudbuild.yaml

```diff
@@ -0,0 +1,13 @@
+steps:
```

Comment:
I'm a bit lost on this cloudbuild.yaml vs the one in my_module. Why are they different? I may be missing something obvious, I'm new to cloud builder.

Reply:
I'm still a bit lost on this. From what I gather it looks like you have to build the container from the subfolder, and that yaml in the subfolder creates a base python container. Then the yaml in the root is used with that container you've built to actually run the CI/CD pipeline you want? That may be worth putting in the readme.

Reply:
See below